app/templates/ui/app/search/search.html

```diff
@@ -1,5 +1,5 @@
 <div class="row" ng-if="!ctrl.currentUser">Please log in to see content</div>
-<div class="row" ng-if="ctrl.currentUser">
+<div class="row padtop" ng-if="ctrl.currentUser">
```

Comment:
Can you wrap the entire contents in a new div with class search, and use that to control padding?